### DISCUSSION TITLE
Pixel 4xl volte enable but ims not registered

### DISCUSSION THREAD

**Discussion by KIRITXASUNA** [kyujin-cho/pixel-volte-patch#14] [Q&A] [answered]
Hi I'm using pixel 4xl the volte toggle is enabled but still not working when i tried to call. when i check ims status it said not registered. Can you please make it work for pixel 4xl too

**Comment by VlaDiRO88** [comment, marked as answer]
Check this https://github.com/kyujin-cho/pixel-volte-patch/discussions/5

**Comment by alrubayyi** [comment]
> Hi I'm using pixel 4xl the volte toggle is enabled but still not working when i tried to call. when i check ims status it said not registered. Can you please make it work for pixel 4xl too

Exactly like you, the option is present and activated, but it does not work 4XL pixel

**Reply by arsradu** [reply]
It's not supported.

**Reply by VlaDiRO88** [reply]
> > Hi I'm using pixel 4xl the volte toggle is enabled but still not working when i tried to call. when i check ims status it said not registered. Can you please make it work for pixel 4xl too
> 
> Exactly like you, the option is present and activated, but it does not work 4XL pixel

Enabling VoLTE manually on Pixel 4a/5 requires modifying EFS partition, so this won't work.https://github.com/kyujin-cho/pixel-volte-patch/discussions/5

**Reply by alrubayyi** [reply]
Yes I noticed
 System Utilizing VOLTE Not enabled 
I hope it works on older devices
thank you for your effort
